financial-templates-lib/logger/Logger.js

```diff
@@ -62,7 +62,7 @@ const Logger = winston.createLogger({
     winston.format.json()
   ),
   transports,
-  exitOnError: false
+  exitOnError: true
```

Comment:
What does this do?

Reply:
it should exit the winston process if one of the loggers crash or break so that the process does not freeze up. I tested it and it _seems_ to work but more testing is still needed.

Reply:
Gotcha. That makes sense. +1